packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -100,16 +110,16 @@ router.post(
   })
 );
 
-/**
- * GET /internal/carequality/directory/organization/:oid
+/***
+ * GET /internal/carequality/directory/organization/legacy/:oid
  *
  * Retrieves the organization with the specified OID from the Carequality Directory.
  * @param req.params.oid The OID of the organization to retrieve.
  * @param req.params.getInactive Optional, indicates whether to get the inactive organization(s). If not provided, will fetch active organizations.
  * @returns Returns the organization with the specified OID.
  */
 router.get(
-  "/directory/organization/:oid",
+  "/directory/organization/legacy/:oid",
```

Comment:
For internal why would need to both endpoints? Wouldnt the legacy one be fine?

Reply:
Done.